packages/semantic-ui/package.json

```diff
@@ -68,6 +67,7 @@
     "mini-css-extract-plugin": "^0.4.3",
     "mocha": "^8.3.2",
     "monaco-editor-webpack-plugin": "^1.7.0",
+    "nanoid": "^3.1.23",
```

Comment:
nanoid should be added as a dependency, not a devDependency, for all packages.

Reply:
Done